src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1765,6 +1768,17 @@ static bool processOnChangeConditionForUpdateContext
       }
     }
   }
+  if (covered == true)
```

Comment:
```suggestion
  if (covered)
```

Simpler :)

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/4068/commits/05ce98dfdba3942ab7a41a5bf76b2903ddeb1e72